src/components/ButtonPanel.js

```diff
@@ -1,38 +1,43 @@
 import React from 'react';
 import Button from './Button';
+import '../styles/ButtonPanel.css';
 
-const ButtonPanel = () => (
-  <div>
-    <div className="btn-group">
-      <Button buttonName="AC" />
-      <Button buttonName="+/-" />
-      <Button buttonName="%" />
-      <Button buttonName="÷" />
+const ButtonPanel = () => {
+  const white = 'rgb(224, 224, 224)';
+  const orange = 'rgb(189, 120, 8)';
+  return (
+    <div className="btn-panel">
+      <div className="btn-group">
+        <Button buttonName="AC" color={white} />
+        <Button buttonName="+/-" color={white} />
+        <Button buttonName="%" color={white} />
+        <Button buttonName="÷" color={orange} />
+      </div>
+      <div className="btn-group">
+        <Button buttonName="7" color={white} />
+        <Button buttonName="8" color={white} />
+        <Button buttonName="9" color={white} />
+        <Button buttonName="×" color={orange} />
+      </div>
+      <div className="btn-group">
+        <Button buttonName="4" color={white} />
+        <Button buttonName="5" color={white} />
```

Comment:
I spoke about this in https://github.com/ermin-cahtarevic/react-calculator/pull/3#discussion_r429557668.
👀 👀

Reply:
Milestone #3 was updated.